neurovault/urls.py

```diff
@@ -380,6 +389,7 @@ class CollectionViewSet(mixins.RetrieveModelMixin,
     filter_fields = ('name', 'DOI', 'owner')
     serializer_class = CollectionSerializer
     filter_backends = (DjangoFilterBackend,)
+    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
```

Comment:
Object level permission are already in handy even though I use them indirectly. As the API expands I expect object level permission's way to scale better than equality checks.